bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -8,6 +8,16 @@ metadata:
     certified: "false"
     createdAt: REPLACE_DATE
     description: An operator that enables Windows container workloads on OCP
+    features.operators.openshift.io/disconnected: "false"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/proxy-aware: "true"
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "false"
```

Comment:
don't we build, ship, and configure cni plugins?

Reply:
yes, thanks for catching. fixed